hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/issues/IssueRegressionTests.java

```diff
@@ -237,133 +229,118 @@ final Stream<DynamicTest> duplicatedTxnsSameTypeDifferentNodesDetected() {
-                                .hasKnownStatus(INVALID_CONTRACT_ID));
+        return hapiTest(
+                uploadInitCode("CreateTrivial"),
+                uploadInitCode("PayReceivable"),
+                cryptoCreate(TRANSFER),
+                contractCreate("CreateTrivial"),
+                contractCreate("PayReceivable"),
+                cryptoDelete(TRANSFER),
+                contractDelete("PayReceivable"),
+                balanceSnapshot(SNAPSHOT, GENESIS),
+                contractDelete("CreateTrivial")
+                        .via(DELETE_TXN)
+                        .transferAccount(TRANSFER)
+                        .hasKnownStatus(OBTAINER_DOES_NOT_EXIST),
+                contractDelete("CreateTrivial")
+                        .via(DELETE_TXN)
+                        .transferContract("PayReceivable")
+                        .hasKnownStatus(INVALID_CONTRACT_ID));
     }
 
     @HapiTest
     final Stream<DynamicTest> multiKeyNonPayerEntityVerifiedAsync() {
         KeyShape LARGE_THRESH_SHAPE = KeyShape.threshOf(1, 10);
         SigControl firstOnly = LARGE_THRESH_SHAPE.signedWith(sigs(ON, OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF));
 
-        return defaultHapiSpec("MultiKeyNonPayerEntityVerifiedAsync")
-                .given(
-                        newKeyNamed("payerKey").shape(LARGE_THRESH_SHAPE),
-                        newKeyNamed("receiverKey").shape(LARGE_THRESH_SHAPE),
-                        cryptoCreate(PAYER).keyShape(LARGE_THRESH_SHAPE),
-                        cryptoCreate(RECEIVER).keyShape(LARGE_THRESH_SHAPE).receiverSigRequired(true))
-                .when()
-                .then(cryptoTransfer(tinyBarsFromTo(PAYER, RECEIVER, 1L))
+        return hapiTest(
+                newKeyNamed("payerKey").shape(LARGE_THRESH_SHAPE),
+                newKeyNamed("receiverKey").shape(LARGE_THRESH_SHAPE),
+                cryptoCreate(PAYER).keyShape(LARGE_THRESH_SHAPE),
+                cryptoCreate(RECEIVER).keyShape(LARGE_THRESH_SHAPE).receiverSigRequired(true),
+                cryptoTransfer(tinyBarsFromTo(PAYER, RECEIVER, 1L))
                         .sigControl(forKey(PAYER, firstOnly), forKey(RECEIVER, firstOnly)));
     }
 
     @HapiTest
     final Stream<DynamicTest> discoversExpectedVersions() {
-        return defaultHapiSpec("discoversExpectedVersions")
-                .given()
-                .when()
-                .then(getVersionInfo().logged().hasNoDegenerateSemvers());
+        return hapiTest(getVersionInfo().logged().hasNoDegenerateSemvers());
     }
 
     @HapiTest
     final Stream<DynamicTest> idVariantsTreatedAsExpected() {
-        return defaultHapiSpec("idVariantsTreatedAsExpected")
-                .given()
-                .when()
-                .then(sendModified(withSuccessivelyVariedQueryIds(), QueryVerbs::getVersionInfo));
+        return hapiTest(sendModified(withSuccessivelyVariedQueryIds(), QueryVerbs::getVersionInfo));
     }
 }
```

Comment:
I wonder why some tests get put here in `issues/` instead of just being in their proper place (functionally speaking)?  Just, history? Tradition?

Reply:
In theory it could be a good idea, but I like grouping by functionality much better